### ISSUE TITLE
Add JHOVE checks to do MD curation

### ISSUE THREAD

**Issue by stranak** [EUDAT-B2SHARE/b2share#164]
Created by Peter Wittenburg

**Comment by thijscobben** [comment]
http://jhove.openpreservation.org/getting-started/

**Comment by thijscobben** [comment]
Is JHOVE about MD curation or about automated data validation on data/digital-objects themselves? The latter seems to be the case. More info on the source of this issue seems required. Until then I would like to mark it INVALID (not implementable/info needed).

**Comment by nharraud** [comment]
"info needed" is fine I think. This seems to have been requested only once by a user community. Thus it will probably have a low priority for the next budget. "invalid" might be too strong as this is still a valid request. If we don't intend to do it at all we can mark it as "WontFix".
